devices/ikea.js

```diff
@@ -625,10 +625,16 @@ module.exports = [
         model: 'E2007',
         vendor: 'IKEA',
         description: 'STARKVIND air purifier',
-        exposes: [e.fan().withModes(['off', 'low', 'medium', 'high', 'auto'])],
+        exposes: [
+            e.fan().withModes(['off', 'low', 'medium', 'high', 'auto']),
+            e.pm25().withAccess(ea.STATE_GET),
+            exposes.enum('air_quality', ea.STATE_GET, [
+                'good', 'ok', 'not_good', 'unknown',
+            ]).withDescription('Measured air quality'),
+        ],
         meta: {fanStateOn: 'auto'},
-        fromZigbee: [fz.fan],
-        toZigbee: [tz.fan_mode],
+        fromZigbee: [fz.fan, fz.ikea_pm25],
+        toZigbee: [tz.fan_mode, tz.ikea_pm25],
         configure: async (device, coordinatorEndpoint, logger) => {
```

Comment:
Since the device does not report these values on its own, I think it is good to poll them (but not too often, maybe every 30 mins?). An example how to setup polling: https://github.com/Koenkk/zigbee-herdsman-converters/blob/04fc9d85c99f2488440d3b8d50e08ed0fc22a26c/devices/tuya.js#L1032 (make sure to also add an option for it https://github.com/Koenkk/zigbee-herdsman-converters/blob/04fc9d85c99f2488440d3b8d50e08ed0fc22a26c/devices/tuya.js#L1028 ). This option will be automagically generated in the docs so no need to further explain that this value is polled.

Reply:
I'm leaning against not doing that, the value will be unknown when the fan is not running anyway.

Reply:
I see, that is fine